packages/extension/src/companion/CompanionMenu.tsx

```diff
@@ -60,14 +60,14 @@ export default function CompanionMenu({
     'companion_helper',
     companionHelper,
   );
-  const updatePost = async (update) => {
+  const updatePost = async ({ update, event }) => {
     const oldPost = post;
     setPost({
       ...post,
       ...update,
     });
     trackEvent(
-      postAnalyticsEvent(postEventName(update), post, {
+      postAnalyticsEvent(event, post, {
         extra: { origin: 'companion context menu' },
```

Comment:
Best to use `Origin.{origin}` here from the enum export

Reply:
Added in c3f306058d206e9b4c8efb75c6f2a4ec1c208c46